packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -361,7 +443,11 @@ export function MenuItem(props: MenuItemProps): React.ReactElement {
 			onKeyDown={handleKeyDown}
 			tabIndex={0}
 		>
-			{text}
+			<div className={styles.itemContent}>
+				{hasChanges && <div className={styles.changeIndicator} />}
```

Comment:
I think we should reconsider the visuals here a bit. I'm not (personally) a fan of the text jumping to the right when the notification appears.

There are a couple of things we could do:
1. Always leave space for the change indicator so that nothing shifts when `hasChanges` changes.
2. Consider an alternative visual queue. A border would work without shifting things.

Reply:
We should also consider making the notification temporary. I.e. have it animate and fade after some time. But we'll need to be careful to make the duration long enough that we don't end up with rapid blinking. Maybe 5 seconds or something?

Second opinions are welcome. I am not a UI expert 😅

Reply:
Second opinions are welcome. I am not a UI expert 😅

Reply:
I still think the red icon to the left of the item is stright foward to the users. I agree with yout first point, where the shift is ugly. It's fixed now and the text should stay in its position.

Reply:
Applied! Now it's rendered for a second.